frontend/src/pages/LandlordPage.tsx

```diff
@@ -115,11 +126,14 @@ const LandlordPage = (): ReactElement => {
   const { landlordId } = useParams<Record<string, string | undefined>>();
   const [landlordData] = useState(dummyData);
   const [aveRatingInfo] = useState(dummyRatingInfo);
-
+  const [reviewData, setReviewData] = useState<Review[]>(reviews);
```

Comment:
We don't relaly have a need for the constant / hardcoded data for reviews anymore because of the work done in this PR. Instead, the initial value of this state can just be `[]`